Sources/Datasets/Dataset.swift

```diff
@@ -24,4 +24,8 @@ public struct S5TFDatasetInfo {
     var version: String
     var description: String
     var homepage: URL
+    var numTrain: Int = 0
+    var numValid: Int = 0
+    var numTest: Int = 0
+    var numFeatures: Int = 0
```

Comment:
Let's wait for this: https://github.com/s5tf-team/datasets/pull/14#discussion_r370927368.

Anyway, `numberOf` is preferred over `num` in Swift. Consider changing `train` to `TrainingExamples` as well.